### ISSUE TITLE
Docker is mandatory?

### ISSUE THREAD

**Issue by dixnor** [harperreed/google-home-notifier-python#2]
This is not an issue more a question:
Can we just run the notifier webservice over Apache. I already have a full home automation solution that already runs an Apache web server. 
Please give us a way to integrate on existing web servers.
Thanks

**Comment by harperreed** [comment]
Nope. Docker isn't a requirement - but it is a good way to separate out all your services. This is just a flask app and could run anywhere
